docs/workers.md

```diff
@@ -589,15 +589,15 @@ regular expressions:
 
     ^/_matrix/client/(r0|v3|unstable)/keys/upload
 
-If `use_presence` is False in the homeserver config, it can also handle REST
+If `use_presence` is set to `false` in the homeserver config, it can also handle REST
 endpoints matching the following regular expressions:
 
     ^/_matrix/client/(api/v1|r0|v3|unstable)/presence/[^/]+/status
 
 This "stub" presence handler will pass through `GET` request but make the
 `PUT` effectively a no-op.
 
-It will proxy any requests it cannot handle to the main synapse instance. It
+It will proxy not cached requests to the main synapse instance. It
```

Comment:
Not sure about this change, but I think `uncached` sounds more natural than `not cached`.